src/globus_sdk/_testing/data/flows/_common.py

```diff
@@ -1,9 +1,17 @@
 from __future__ import annotations
 
 import typing as t
+import uuid
 
 TWO_HOP_TRANSFER_FLOW_ID = "24bc4997-b483-4c25-a19c-64b0afc00743"
 TWO_HOP_TRANSFER_FLOW_OWNER_ID = "b44bddda-d274-11e5-978a-9f15789a8150"
+
+FLOW_ID = str(uuid.uuid1())
+RUN_ID = str(uuid.uuid1())
+USER1 = f"urn:globus:auth:identity:{uuid.uuid1()}"
+USER2 = f"urn:globus:auth:identity:{uuid.uuid1()}"
+GROUP = f"urn:globus:groups:id:{uuid.uuid1()}"
```

Comment:
Pure curiosity: why use uuid v1?

Reply:
I generally only use v1 and v4 (because I never have needed the special namespacing features of v5).
Between v1 and v4 I don't really have much preference, but v1 is very slightly faster to generate (fewer random bits) so I tend towards it absent any particular reason to use v4 for test data, etc.